packages/typegpu/src/tgsl/wgslGenerator.ts

```diff
@@ -711,18 +707,17 @@ ${alternate}`;
         [condSnippet] = converted;
       }
     }
-    const conditionStr = condSnippet ? ctx.resolve(condSnippet.value) : '';
+    const conditionStr = condSnippet
+      ? ctx.resolve(condSnippet.value).trim()
```

Comment:
We add whitespace, then we remove whitespace. Could we acomplish this without `.trim()`? 👀